data/scripts/talkactions/god/add_mount.lua

```diff
@@ -16,23 +10,41 @@ function addOutfit.onSay(player, words, param)
 	end
 
 	local split = param:split(",")
-	local name = split[1]
-
-	local target = Player(name)
-	if target then
-		local mount = tonumber(split[2])
-		target:addMount(mount)
-		target:sendTextMessage(MESSAGE_ADMINISTRATOR, "" .. player:getName() .. " has been added a new mount for you.")
-		player:sendTextMessage(MESSAGE_ADMINISTRATOR, "You have sucessfull added mount " .. mount .. " to the player " .. target:getName() .. ".")
-		if printConsole then
-			logger.info("[addOutfit.onSay] - Player: {} has been added mount: {} to the player: {}", player:getName(), lookType, target:getName())
-		end
+	if #split < 2 then
+		player:sendCancelMessage("Usage: /addmount <playername>, <mount id or 'all'>")
+		return true
+	end
+
+	local playerName = split[1]
+	local target = Player(playerName)
+
+	if not target then
+		player:sendCancelMessage("Player not found.")
 		return true
 	end
-	player:sendCancelMessage("Player not found.")
+
+	local mountParam = string.trim(split[2])
+	if mountParam == "all" then
+		for mountId = 1, 231 do
```

Comment:
Same here, but it's even easier, you can just export "getMounts().size()" and call it here

```cpp

int GameFunctions::luaGameGetMountsSize(lua_State* L) {
	// Game.getMountsSize()
	lua_pushnumber(L, g_game().getMounts().size());
	return 1;
}
```